draft-packages/form/KaizenDraft/Form/Primitives/Input/styles.scss

```diff
@@ -125,6 +125,11 @@ $input-with-icon-padding: $input-icon-size + $ca-grid * 0.75;
   .endIconAdornment {
     @include vertically-center-icon;
     @include ca-position($end: $ca-grid / 2);
+
+    &.datePicker {
```

Comment:
I was able to replicate the styles required without the change to the primitive.

In `DateInput.scss` `.iconButton`, add:
```
  right: -0.25rem;
  top: -45%;
```
and you'll be able to remove all the changes in this component :)

Reply:
Awesome! I was about to also look into this because we would need to update this later for RTL so a lot better its contained